charts/tidb-lightning/values.yaml

```diff
@@ -95,9 +95,15 @@ backend: importer # importer | tidb
 # Specify a Service Account for lightning
 # serviceAccount:
 
-# For TiDB-Lightning v3.0.18+ or v4.0.3+,
-# if we want to log to stdout, we should set `file = "-"`.
+# For TiDB-Lightning v3.0.18+ or v4.0.3+, if you want to log to stdout, you should set `file = "-"`.
+# If you want to store the checkpoint under source data directory,
+# you should set `CHECKPOINT_USE_DATA_DIR` as the prefix of `dsn`.
```

Comment:
We should make it clear that if `CHECKPOINT_USE_DATA_DIR` is not changed in this file, it will be set to the which dir for different modes, e.g., local, ad-hoc, and remote.

Reply:
add some comments in https://github.com/pingcap/tidb-operator/pull/3653/commits/fdf1378b8b4ba9ea3afe32db9b5ec298c2499bfb.